src/components/Form.js

```diff
@@ -39,19 +33,23 @@ const AddNewBook = () => {
   const submitBookToStore = (e) => {
     e.preventDefault();
     const newBook = {
-      id: store.getState().bookReducer.length,
+      item_id: uuidv4(),
       title: form.inputTitle,
-      author: form.inputAuthor,
+      author: 'Toyosi',
       category: form.selectedText,
     };
 
-    dispatch(addBook(newBook));
+    dispatch(postBook(newBook));
   };
```

Comment:
**[OPTIONAL]** After creating and submitting a new book it's a good user experience to clear both input fields to allow the user to enter the new book in the fresh input field.

Reply:
Thanks for the suggestions 👍  @Mre55